manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -150,6 +150,14 @@ spec:
               value: "true"
             {{- end }}
             {{- end }}
+            - name: GOMEMLIMIT
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.memory
+            - name: GOMAXPROCS
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.cpu
```

Comment:
What's this value if cpu/mem limits are not specified

Reply:
0 which go will treat the same as unset

Reply:
it isn't unfortunately, it fails instead https://github.com/istio/istio/pull/46253#issuecomment-1898985902 at least with eks 1.27

Reply:
correction: it's more complicated than that, it conflicts with env but if no other env, it's fine (well not that fine because unset would be reverting to guessing the host machine cpu, so imo 2 as a default is better)